### PR TITLE
[kernel]Normalized kernel API annotation

### PR DESCRIPTION
**Pull request by yangjie11**
## 拉取/合并请求描述：(PR description)

[
内核注释规范化
]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [x] 本拉取/合并使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](../documentation/coding_style_cn.md) This PR complies with [RT-Thread code specification](../documentation/coding_style_en.txt)

### CODE REVIEW

File: src/device.c [hunk 1]
```diff
@@ -134,9 +137,9 @@ rt_device_t rt_device_create(int type, int attach_size)
 RTM_EXPORT(rt_device_create);
 
 /**
- * This function destroy the specific device object.
+ * @brief This function destroy the specific device object.
  *
- * @param dev, the specific device object.
+ * @param dev the specific device object.
```
Comment (mysterywolf) [inline src/device.c:142]: dev is a specific device object.

File: src/device.c [hunk 2]
```diff
@@ -153,11 +156,11 @@ RTM_EXPORT(rt_device_destroy);
 #endif /* RT_USING_HEAP */
 
 /**
- * This function will initialize the specified device
+ * @brief This function will initialize the specified device.
  *
- * @param dev the pointer of device driver structure
+ * @param dev the pointer of device driver structure.
```
Comment (mysterywolf) [inline src/device.c:161]: dev is the pointer of device driver structure.

File: src/device.c [hunk 3]
```diff
@@ -72,9 +74,9 @@ rt_err_t rt_device_register(rt_device_t dev,
 RTM_EXPORT(rt_device_register);
 
 /**
- * This function removes a previously registered device driver
+ * @brief This function removes a previously registered device driver.
  *
- * @param dev the pointer of device driver structure
+ * @param dev the pointer of device driver structure.
```
Comment (mysterywolf) [inline src/device.c:79]: 缺be动词 param的都存在这个问题。

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: src/device.c [hunk 4]
```diff
@@ -39,11 +39,13 @@
 #endif /* RT_USING_DEVICE_OPS */
 
 /**
- * This function registers a device driver with specified name.
+ * @brief This function registers a device driver with specified name.
```
Comment (mysterywolf) [inline src/device.c:42]: with a specified name.

File: src/device.c [hunk 5]
```diff
@@ -391,13 +403,14 @@ rt_err_t rt_device_control(rt_device_t dev, int cmd, void *arg)
...
+ *        is invoked when this device receives data.
+ *
+ * @param dev the pointer of device driver structure.
  *
- * @param dev the pointer of device driver structure
- * @param rx_ind the indication callback function
+ * @param rx_ind the indication callback function.
  *
- * @return RT_EOK
+ * @return RT_EOK.
```
Comment (mysterywolf) [inline src/device.c:413]: 这个不用加句号 不是一个句子

File: src/device.c [hunk 6]
```diff
@@ -413,13 +426,14 @@ rt_device_set_rx_indicate(rt_device_t dev,
...
+ *        written data to physical hardware.
+ *
+ * @param dev the pointer of device driver structure.
  *
- * @param dev the pointer of device driver structure
- * @param tx_done the indication callback function
+ * @param tx_done the indication callback function.
  *
- * @return RT_EOK
+ * @return RT_EOK.
```
Comment (mysterywolf) [inline src/device.c:436]: 不用加句号

File: src/scheduler.c [hunk 7]
```diff
@@ -407,8 +411,8 @@ void rt_schedule(void)
 }
 #else
 /**
- * This function will perform one schedule. It will select one thread
- * with the highest priority level, and switch to it immediately.
+ * @brief This function will perform one schedule. It will select one thread
```
Comment (mysterywolf) [inline src/scheduler.c:414]: scheduling

File: src/scheduler.c [hunk 8]
```diff
@@ -631,11 +635,12 @@ void rt_scheduler_do_irq_switch(void *context)
 }
 #endif /* RT_USING_SMP */
 
-/*
- * This function will insert a thread to system ready queue. The state of
- * thread will be set as READY and remove from suspend queue.
+/**
+ * @brief This function will insert a thread to system ready queue. The state of
+ *        thread will be set as READY and remove from suspend queue.
```
Comment (mysterywolf) [inline src/scheduler.c:640]: remove from the suspend queue.
冠词这个审查的比较松，因为冠词的缺失不会造成英语母语者的阅读困难，但是也要注意冠词的问题。

**Review by mysterywolf — CHANGES_REQUESTED** [review]

**Review by mysterywolf — APPROVED** [review]

**Comment by BernardXiong** [comment]
@Guozhanxin Please check it and whether +2.

**Review by Guozhanxin — APPROVED** [review]

**Pull request merged**